src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -67,8 +73,19 @@ public LauncherSubsystem() {
 		launcherAngleEncoder = launcherAngleMotor.getAbsoluteEncoder(Type.kDutyCycle);
 
 		// PID controllers
+		// Create launcherTopPIDController and launcherTopMotor]
+		launcherTopPIDController = launcherTopMotor.getPIDController();
+		launcherTopPIDController.setFeedbackDevice(launcherTopEncoder);
+		launcherBottomPIDController = launcherBottomMotor.getPIDController();
+		launcherBottomPIDController.setFeedbackDevice(launcherBottomEncoder);
```

Comment:
These need to be initialized in the `configMotors()` method below so they start from known values. Do that by calling `setP`, `setI`, `setD`, `setFF` on each of these with new p, i, d, ff constants defined above. I expect the same values can be used for both the top & bottom motors.

Reply:
done 👍